src/connector/src/connector_common/connection.rs

```diff
@@ -132,15 +132,14 @@ pub struct IcebergConnection {
     #[serde(rename = "gcs.credential")]
     pub gcs_credential: Option<String>,
 
-    /// Path of iceberg warehouse, only applicable in storage catalog.
+    /// Path of iceberg warehouse.
     #[serde(rename = "warehouse.path")]
     pub warehouse_path: Option<String>,
     /// Catalog id, can be omitted for storage catalog or when
     /// caller's AWS account ID matches glue id
     #[serde(rename = "glue.id")]
     pub glue_id: Option<String>,
-    /// Catalog name, can be omitted for storage catalog, but
-    /// must be set for other catalogs.
+    /// Catalog name, default value is risingwave.
```

Comment:
I'm a little confused about the default value though. Will user really create a catalog called risingwave? 🤔

Reply:
Actually not every catalog needs a catalog name to access it, so for these catalogs, we don't require users to provide us a catalog name. For those catalog required a catalog name like `JdbcCatalog`, they should set the catalog name. If not, we will use `risingwave` as the catalog. (Although `JdbcCatalog` would use `jdbc` as catalog name if it is unspecified).